### DISCUSSION TITLE
Identus discussions kick-off

### DISCUSSION THREAD

**Discussion by elribonazo** [hyperledger-identus/hyperledger-identus#69] [Announcements]
## 👋 Welcome Community!
  We're proposing to use Discussions as a place to connect and engage with the community in a more collaborative way with the objective of:
  * Having honest and transparent talks about product roadmap, features
  * Creating a shared knowledge base around our technology stack
  * Engaging with other projects and new ideas inside **Identus community**
  * Collectively driving this community growth by being more transparent, inclusive and supportive
  
As part of this shift in mindset and as we announced in previous **Hyperledger community** calls, we are going to be shifting some of our priorities in the following days. Some of the tasks and tickets inside the [Project](https://github.com/orgs/hyperledger/projects/48) are going to change as we are no longer going to prioritize them.

The **Identus team in IOE** is going to have main focus on:
1. Making infra easier to deploy, cheaper and maintain over time
2. Reduce complexity in the implementation
3. Improve support, guides and documentation
4. Interoperability with other systems (DIDs, VDR) 
5. Integrating other tools on **IOHK**, (Lace, Midnight)
6. Collectively building long term governance framework around **Identus**  
  
What outcome we expect from this shift?
* Clear roadmap is defined
* Establish virtual working groups (in github discussions) and kickoff product and feature talks
* All projects and **Identus** members will have a space to bring topics and open debate
* All doors are open, we'd love to see new contributions coming from the community
* Improvements and more clarity in terms of Governance framework

**Comment by mixmix** [comment]
This is great, thanks for sharing about the future of Identus @elribonazo . The [Matou](www.matou.nz) team is here to grow this. We look forward to collaborating on this in into the future!

> All projects and Identus members will have a space to bring topics and open debate

Do you have an idea of what an "Identus member" is? Perhaps that could a topic of discussion in itself

**Reply by elribonazo** [reply]
Identus member, anyone in here that has been interacting with the community either by involvement in Hyperledger Community calls, contributing to projects, discussions, etc. Anyone really.

Identus maintainers is another discussion really, those in my opinion are just profiles that have had active involvement in the project for a long enough period by concretely making contributions, improvements to the documentation. Identus maintainers have a higher degree of responsability in the project cause they take some decisions on what can be merged so those are more like longer term relationships if it makes sense
